src/System.Net.Http/tests/StressTests/HttpStress/Program.cs

```diff
@@ -386,17 +405,19 @@ void ValidateContent(string expectedContent, string actualContent)
             clientOperations = opIndices.Select(i => clientOperations[i]).ToArray();
         }
 
-        Console.WriteLine("       .NET Core: " + Path.GetFileName(Path.GetDirectoryName(typeof(object).Assembly.Location)));
-        Console.WriteLine("    ASP.NET Core: " + Path.GetFileName(Path.GetDirectoryName(typeof(WebHost).Assembly.Location)));
-        Console.WriteLine("         Tracing: " + (logPath == null ? (object)false : logPath.Length == 0 ? (object)true : logPath));
-        Console.WriteLine("     ASP.NET Log: " + aspnetLog);
-        Console.WriteLine("     Concurrency: " + concurrentRequests);
-        Console.WriteLine("  Content Length: " + maxContentLength);
-        Console.WriteLine("   HTTP Versions: " + string.Join<Version>(", ", httpVersions));
-        Console.WriteLine("        Lifetime: " + (connectionLifetime.HasValue ? $"{connectionLifetime}ms" : "(infinite)"));
-        Console.WriteLine("      Operations: " + string.Join(", ", clientOperations.Select(o => o.Item1)));
-        Console.WriteLine("     Random Seed: " + seed);
-        Console.WriteLine("Query Parameters: " + numParameters);
+        Console.WriteLine("             .NET Core: " + Path.GetFileName(Path.GetDirectoryName(typeof(object).Assembly.Location)));
+        Console.WriteLine("          ASP.NET Core: " + Path.GetFileName(Path.GetDirectoryName(typeof(WebHost).Assembly.Location)));
+        Console.WriteLine("               Tracing: " + (logPath == null ? (object)false : logPath.Length == 0 ? (object)true : logPath));
+        Console.WriteLine("           ASP.NET Log: " + aspnetLog);
+        Console.WriteLine("           Concurrency: " + concurrentRequests);
+        Console.WriteLine("        Content Length: " + maxContentLength);
+        Console.WriteLine("         HTTP Versions: " + string.Join<Version>(", ", httpVersions));
+        Console.WriteLine("              Lifetime: " + (connectionLifetime.HasValue ? $"{connectionLifetime}ms" : "(infinite)"));
+        Console.WriteLine("            Operations: " + string.Join(", ", clientOperations.Select(o => o.Item1)));
+        Console.WriteLine("           Random Seed: " + seed);
+        Console.WriteLine("      Query Parameters: " + numParameters);
+        Console.WriteLine("Client Max Buffer Size: " + clientMaxBufferSize);
+        Console.WriteLine("    Server Buffer Size: " + serverBufferSize);
```

Comment:
It's strange to have options here that don't apply to all operations.  I don't think we should have separate client operations for this: if we want to control this, that's fine, but the controls should apply to the data sent and received by all of these operations.